### PR TITLE
Document linter usage for developers

### PR DESCRIPTION
**Pull request by mjgiarlo**
No surprises.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,9 +54,19 @@ RSpec.describe 'Create a work draft', :rack_test do
...
+### Code Linters
+
+To run all configured linters, run `bin/rake lint`.
+
+To run linters individually, run which ones you need:
+
+* Ruby code: `bin/rubocop`
+* ERB templates: `bin/erb_lint --lint-all --format compact`
+* JavaScript code: `yarn run lint`
+
```
Comment (justinlittman) [inline README.md:69]: Can we also document the format for autofixing (since that is what we will mostly be running)?

Comment (mjgiarlo) [reply README.md:69]: Done.

File: app/javascript/application.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
-import "@hotwired/turbo-rails"
-import "controllers"
-import * as bootstrap from "bootstrap"
+import '@hotwired/turbo-rails'
+import 'controllers'
+import * as bootstrap from 'bootstrap' // eslint-disable-line no-unused-vars
```
Comment (mjgiarlo) [inline app/javascript/application.js:4]: Invoking the JS linter outside the context of `yarn run`, it found additional violations. That and the fact that we don't need yarn scripts to run linters motivated me to change our approach to JS linting ever-so-slightly.

File: package.json [hunk 3]
```diff
@@ -16,11 +16,15 @@
     "build:css:compile": "sass ./app/assets/stylesheets/application.bootstrap.scss:./app/assets/builds/application.css --no-source-map --load-path=node_modules",
     "build:css:prefix": "postcss ./app/assets/builds/application.css --use=autoprefixer --output=./app/assets/builds/application.css",
     "build:css": "yarn build:css:compile && yarn build:css:prefix",
-    "watch:css": "nodemon --watch ./app/assets/stylesheets/ --ext scss --exec \"yarn build:css\"",
-    "lint": "standard app/javascript/**/*.js",
-    "lint_fix": "standard --fix app/javascript/**/*.js"
```
Comment (mjgiarlo) [inline package.json:21]: Unnecessary indirection, I might argue.

Comment (mjgiarlo) [reply package.json:21]: And now I disagree because CI depends on it.

File: Rakefile [hunk 4]
```diff
@@ -15,14 +15,14 @@ begin
...
   task erblint: :environment do
-    puts 'Running erblint...'
+    puts 'Running ERB linter...'
     system('bin/erb_lint --lint-all --format compact')
   end
 
   desc 'Run Yarn linter against JS files'
   task eslint: :environment do
-    puts 'Running JS linters...'
-    system('yarn run lint')
```
Comment (mjgiarlo) [inline Rakefile:25]: The `run` isn't necessary 🤷🏻‍♂️

File: app/javascript/application.js (hunk 2 above)
Comment (mjgiarlo) [inline app/javascript/application.js:4]: The changes to how we invoke `standard` in `package.json` found some violations we had been missing before.

**Review by justinlittman — APPROVED** [review]

**Pull request merged**